madgraph.sh

```diff
@@ -0,0 +1,28 @@
+package: madgraph
+version: "%(tag_basename)s"
+tag: "v2.8.1"
+source: https://github.com/mfasDa/MadGraph5
```

Comment:
Please do not use a personal repository.

Reply:
As mentioned above someone with rights needs to create the repository. @preghenella @maireiphc @ktf 

Names could be 
madgraph
madgraph5
MadGraph
MadGraph5

I will adapt to what fits best to the naming convention.

Reply:
request to open the repo opened.
https://alice.its.cern.ch/jira/browse/O2-1754

@mfasDa 
is a brand new repo needed? or can you fork from another repo?

Reply:
Unfortunately they use launchpad, so no git repo we can fork from :(

Reply:
the repo is ready, you can fork it, populate it and open PR.
Please, ask in the PR for the tag if it needs to be created.

Reply:
@preghenella  Forking is prevented while the repository is still empty - I suspect you need to import it from my repository.

Reply:
uhm... can we give you the write rights in this repo?

Reply:
@mfasDa  I have started the import from your repository.

Reply:
done @mfasDa 
you also have the tag ready there, check it out!
https://github.com/alisw/MadGraph

Reply:
OK, done as well. The build recipe is using the tag from the alisw repo